roles/rsyslog/roles/output_roles/files/defaults/main.yaml

```diff
@@ -39,7 +39,7 @@ __rsyslog_conf_files_output_rules:
     sections:
 
       - options: |-
-          {% if rsyslog_files_actions is defined and rsyslog_files_actions | length > 0 %}
+          {% if rsyslog_files_actions is defined and rsyslog_files_actions %}
```

Comment:
would
```
rsyslog_files_actions | d([])
```
work? @richm what do you think?

Reply:
Fixed.   I also fixed similar codes in roles/rsyslog/tasks/main.yml.